### PR TITLE
AO3-4772 Extend tests for tag_set_nominations_controller.rb

### PR DESCRIPTION
**Pull request by pixelred**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4772

sorry about the onslaught of meaningless commit messages... it took a lot of minor tweaking before i finally got it to work

### CODE REVIEW

File: features/tags_and_wrangling/tag_set.feature [hunk 1]
```diff
@@ -79,15 +79,190 @@ Feature: creating and editing tag sets
...
+  Then I should see "Your nominations were successfully submitted"
+  Given I am logged in as "tagsetter"
+  When I review nominations for "Nominated Tags"
+    And I follow "Clear Nominations"
+    And I press "Yes, Clear Tag Set Nominations"
+  Then I should see "All nominations for this Tag Set have been cleared"
+
+  Scenario: Owner of a tag set with over 30 nominations sees a message that they can't all be displayed on one page
+  Given I am logged in as "tagsetter"
+    And I set up the nominated tag set "Nominated Tags" with 6 fandom noms and 6 character noms
```
Comment (zz9pzza) [inline features/tags_and_wrangling/tag_set.feature]: I would look at the method used in https://github.com/otwcode/otwarchive/blob/master/features/step_definitions/admin_steps.rb#L236

Comment (pixelred) [inline features/tags_and_wrangling/tag_set.feature]: ahhh thank you... i was thinking there must be a more intuitive way to do that

**Comment by sarken** [comment]
I just wanted to drop a quick note and point out that you can set up your own Travis following the instructions on this page: https://github.com/otwcode/otwarchive/wiki/Travis-CI (If you've already opened a pull request, you'll have to close it to avoid using the org's Tavis when you push, but they're easily re-opened.)

**Comment by pixelred** [comment]
oh! i have travis set up on mine; i can close this until i've figured out what i'm doing wrong

File: features/step_definitions/tag_set_steps.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@
 # Takes things like When I add the fandom tags "Bandom" to the tag set "MoreJoyDay". Don't forget the extra s, even if it's singular.
 When /^I add (.*) to the tag ?set "([^\"]*)"$/ do |tags, title|
   step %{I go to the "#{title}" tag set edit page}
-    tags.scan(/the (\w+) tags "([^\"]*)"/).each do |type, tags| 
+    tags.scan(/the (\w+) tags "([^\"]*)"/).each do |type, tags|
```
Comment (zz9pzza) [inline features/step_definitions/tag_set_steps.rb]: I would agree with the hound here.

Comment (pixelred) [inline features/step_definitions/tag_set_steps.rb]: about the shadowing or the indentation (or both)?

File: features/step_definitions/tag_set_steps.rb [hunk 3]
```diff
@@ -27,8 +27,8 @@
 # Takes things like When I add the fandom tags "Bandom" to the tag set "MoreJoyDay". Don't forget the extra s, even if it's singular.
 When /^I add (.*) to the tag ?set "([^\"]*)"$/ do |tags, title|
   step %{I go to the "#{title}" tag set edit page}
-    tags.scan(/the (\w+) tags "([^\"]*)"/).each do |type, tags| 
-      fill_in("owned_tag_set_tag_set_attributes_#{type}_tagnames_to_add", :with => tags)
+  tags.scan(/the (\w+) tags "([^\"]*)"/).each do |type, tags|
```
Comment (zz9pzza) [inline features/step_definitions/tag_set_steps.rb]: The shadowing.

File: features/step_definitions/tag_set_steps.rb (hunk 2 above)
Comment (zz9pzza) [inline features/step_definitions/tag_set_steps.rb]: Shadowing

File: features/step_definitions/tag_set_steps.rb (hunk 3 above)
Comment (pixelred) [inline features/step_definitions/tag_set_steps.rb]:
```
When /^I add (.*) to the tag ?set "([^\"]*)"$/ do |tags, title|
  step %{I go to the "#{title}" tag set edit page}
    tags.scan(/the (\w+) tags "([^\"]*)"/).each do |type, scannedtags|
      fill_in("owned_tag_set_tag_set_attributes_#{type}_tagnames_to_add", :with => scannedtags)
```

?

File: features/step_definitions/tag_set_steps.rb (hunk 2 above)
Comment (pixelred) [inline features/step_definitions/tag_set_steps.rb]:
```
When /^I add (.*) to the tag ?set "([^\"]*)"$/ do |tags, title|
  step %{I go to the "#{title}" tag set edit page}
    tags.scan(/the (\w+) tags "([^\"]*)"/).each do |type, scannedtags|
      fill_in("owned_tag_set_tag_set_attributes_#{type}_tagnames_to_add", :with => scannedtags)
```

?

**Comment by pixelred** [comment]
travis seems to have blipped a prompt meme test; but the tag set nominations passed 👍

**Pull request merged**